src/test/ui/macros/macro-in-fn.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(decl_macro)]
+
+pub fn moo() {
+    pub macro ABC() {{}}
+}
+
+fn main() {
+    ABC!(); //~ ERROR cannot find macro `ABC!` in this scope
```

Comment:
This line actually prevents the ICE.
With this error compilation stops after type checking and never reaches the privacy-reachability pass that ICEs.

Reply:
(So it's better to remove it and turn the test into `check-pass`.)